site/content/en/v1.0.0/user/traffic/udp-routing.md

```diff
@@ -1,24 +1,18 @@
 ---
-title: "UDP Routing"
+title: UDP Routing
 ---
 
-The [UDPRoute][] resource allows users to configure UDP routing by matching UDP traffic and forwarding it to Kubernetes
-backends. This guide will use CoreDNS example to walk you through the steps required to configure UDPRoute on Envoy
-Gateway.
+The [UDPRoute](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1alpha2.UDPRoute) resource allows users to configure UDP routing by matching UDP traffic and forwarding it to Kubernetes backends. This guide will use CoreDNS example to walk you through the steps required to configure UDPRoute on Envoy Gateway.
 
-__Note:__ UDPRoute allows Envoy Gateway to operate as a non-transparent proxy between a UDP client and server. The lack
-of transparency means that the upstream server will see the source IP and port of the Gateway instead of the client.
-For additional information, refer to Envoy's [UDP proxy documentation][].
+**Note:** UDPRoute allows Envoy Gateway to operate as a non-transparent proxy between a UDP client and server. The lack of transparency means that the upstream server will see the source IP and port of the Gateway instead of the client. For additional information, refer to Envoy's [UDP proxy documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/listeners/udp_filters/udp_proxy).
 
 ## Prerequisites
 
-Follow the steps from the [Quickstart](../../quickstart) guide to install Envoy Gateway and the example manifest.
-Before proceeding, you should be able to query the example backend using HTTP.
+Follow the steps from the [Quickstart](../quickstart) guide to install Envoy Gateway and the example manifest. Before proceeding, you should be able to query the example backend using HTTP.
```

Comment:
this is  not right I think, quickstart is outside of traffic dir

Reply:
Checking it manually is hard to maintain, do not wish to bring more broken links

Reply:
that's why istio/eg use linkinator, it will help at the most of cases.